ui/src/app/kvm/views/KVMList/KVMListHeader/KVMActionFormWrapper/RefreshForm/RefreshForm.tsx

```diff
@@ -1,8 +1,9 @@
-import React from "react";
```

Comment:
Looks like this and the Delete form could possibly extend a common component.

Reply:
I think what I'll end up doing is creating a common component that can be used for all the "action forms", including the machine list forms. But I don't think it makes sense for only the KVM list forms since there's only two.

Reply:
Sounds good.